src/mds/Server.cc

```diff
@@ -5823,6 +5823,34 @@ void Server::handle_set_vxattr(MDRequestRef& mdr, CInode *cur)
     auto pi = cur->project_inode(mdr);
     cur->setxattr_ephemeral_dist(val);
     pip = pi.inode.get();
+  } else if (name == "ceph.dir.rctime") {
+    if (!cur->is_dir()) {
```

Comment:
How do you update an inode's rctime? You could fix all of the rctimes but none of the inodes?

Reply:
`project_rstat_frag_to_inode()` and `project_rstat_inode_to_frag()` help with that